tests/data/dual-intel-xeon-gold-6126.xml

```diff
@@ -0,0 +1,110 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Add case of `xavier` (ARMv8 Processor rev 0 (v8l)).

Reply:
Indeed. We added this case. And one for the Apple M2 too. :factory: